examples/Example14/CommonStruct.h

```diff
@@ -29,12 +29,18 @@ struct TrackData {
       : position{x, y, z}, direction{dirx, diry, dirz}, energy{ene}, pdg{pdg_id}
   {
   }
+  
+  inline double FullSort()
+  {
+    return pdg * energy * position[0] * position[1] * position[2] * direction[0] * direction[1] * direction[2];
+  }
```

Comment:
This isn't very robust, if one of the value happens to be zero the product will be as well. Why not implement a proper `operator<`?

Reply:
I wanted to test different sorting criteria, so it can stay as a lambda for a while. But the problem with 0 is indeed an issue, I'll fix this

Reply:
But why the multiplication? `operator<` can compare a number of members one after the other.